lib/edit/s_info.txt

```diff
@@ -394,22 +394,11 @@ E:Thaumaturgy:Antimagic
 
 ################################## SNEAKINESS SKILLS ###############################
 
-N:35:Sneakiness
```

Comment:
It'll be good to get these useless skills out of the RANDOM_GAIN pool... though I'm a little worried that we might end up in a situation where there end up being only "magic" skills -- so that an Anti-magic character will end up with no viable (but uselses) choices. The proper fix to that would of course be to change the algorithm which picks the random skills to choose from so that that doesn't happen.

Just something to be wary of, I guess. (I can certainly implement that change if this goes in.)